compiler/luci/pass/src/VerifyQuantizedNodeType.h

```diff
@@ -129,9 +131,12 @@ class VerifyQuantizedNodeTypeBase : public luci::CircleNodeVisitor<bool>,
 
   // NOTE below nodes has differnent implementation for Qtype/Btype and
   //      implementations exist in VerifyQuantizedNodeU8Type, VerifyQuantizedNodeS16Type
-  // bool visit(const luci::CircleLogistic *node);
-  // bool visit(const luci::CircleSoftmax *node);
-  // bool visit(const luci::CircleTanh *node);
+  // NOTE adding dummy visit() to make compiler happy with "-Werror=overloaded-virtual="
+  // clang-format off
+  bool visit(const luci::CircleLogistic *) { assert(false); return false; }
+  bool visit(const luci::CircleSoftmax *) { assert(false); return false; }
+  bool visit(const luci::CircleTanh *) { assert(false); return false; }
```

Comment:
It might be good to include a description of the cause in the assert message.

```suggestion
  bool visit(const luci::CircleLogistic *) { assert(false && "visit(CircleLogistic) should not be called"); return false; }
  bool visit(const luci::CircleSoftmax *) { assert(false && "visit(CircleSoftmax) should not be called"); return false; }
  bool visit(const luci::CircleTanh *) { assert(false && "visit(CircleTanh) should not be called"); return false; }
```